src/panels/lovelace/cards/hui-gauge-card.ts

```diff
@@ -10,6 +10,7 @@ import isValidEntityId from "../../../common/entity/valid_entity_id.js";
 import { fireEvent } from "../../../common/dom/fire_event.js";
 
 import "../../../components/ha-card.js";
+import { TemplateResult } from "lit-html";
```

Comment:
Minor: keep absolute imports above relative ones unless order matters (in case of jquery/roundslider)